site/docs/transform/sample.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: docs
+menu: docs
+title: Sample
+permalink: /docs/sample.html
+---
+
+The **sample** transform randomly samples a data stream to create a smaller stream. As input data objects are added and removed, the sampled values may change in first-in, first-out manner. This transform uses [reservoir sampling](https://en.wikipedia.org/wiki/Reservoir_sampling) to maintain a representative sample of the stream.
```

Comment:
We don't call it stream in Vega-Lite.

Reply:
> The **sample** transform randomly samples a data stream to create a smaller stream.

The sample transform randomly samples a data source to create a smaller source.
Is this ok?